GenSysPara.m

```diff
@@ -37,7 +37,7 @@
 sysPara.NumElements = 24;       % interger scaler. number of antenna elements
 sysPara.NumChannel = 4;         % interger scaler. number of used channels
 sysPara.Radius = 0.162;         % double scaler. radius of UCA array, in meter. e.g. 3rd Gen = 0.162;
-sysPara.StvIncludeElementResponse = false; % boolen scaler. Include individual element response in the calculation of steering vector
+sysPara.StvIncludeElementResponse = true; % boolen scaler. Include individual element response in the calculation of steering vector
```

Comment:
% boolen scaler. Include individual element response in the calculation of steering vector when generating the received waveforms from channels.